content/news/XXXX-XX-XX-russians-war.md

```diff
@@ -0,0 +1,20 @@
+title: Russian's war on Ukraine
+authors: Daniel Schürmann
+tags: russian ukraine war
+status: draft
+comments: yes
+
+Since the 2022-02-24 we receive horrible news from Ukaine, which have become even worse since then. This requires that we all speak up and show solidarity for Ukraine.
```

Comment:
```suggestion
Since 2022-02-24 we receive horrible news from Ukaine, which have become even worse since then. As an organization commited to freedom we feel the urge speak up and show solidarity for Ukraine.
```